samples/cpp_tokens/slex/cpp_slex_lexer.hpp

```diff
@@ -134,15 +134,20 @@ class lexer
 #define OCTALDIGIT          "[0-7]"
 #define DIGIT               "[0-9]"
 #define HEXDIGIT            "[0-9a-fA-F]"
-#define OPTSIGN             "[-+]?"
+#define BINARYDIGIT         "[01]"
+#define SIGN             "[-+]?"
```

Comment:
Minor, but can we leave this as `OPTSIGN`? More exact, and also this `SIGN` doesn't line up with the other strings